be/src/exec/hash_join_node.cpp

```diff
@@ -485,8 +485,17 @@ pipeline::OpFactories HashJoinNode::_decompose_to_pipeline(pipeline::PipelineBui
     auto&& rc_rf_probe_collector = std::make_shared<RcRfProbeCollector>(2, std::move(this->runtime_filter_collector()));
     // In default query engine, we only build one hash table for join right child.
     // But for pipeline query engine, we will build `num_right_partitions` hash tables, so we need to enlarge the limit
+
+    auto query_options = runtime_state()->query_options();
+
+    size_t global_runtime_filter_build_max_size = UINT64_MAX;
```

Comment:
should INT32_MAX?

Reply:
> should INT32_MAX?

Here we set it to UINT64_MAX to solve the compatibility issue, because the old FE will not transmit `global_runtime_filter_build_max_size`, and we have to ensure that the restriction on partial rf does not take effect during upgrading. If we set it to INT32_MAX, it may still be risky.